cli/azd/pkg/binding/binding_config.go

```diff
@@ -0,0 +1,174 @@
+package binding
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/servicelinker/armservicelinker/v2"
+)
+
+// A child property of `services` in azure.yaml, used to define bindings of a service
+type BindingConfig struct {
+	Name           string             `yaml:"name"`
+	TargetType     TargetResourceType `yaml:"targetType"`
+	TargetResource string             `yaml:"targetResource"`
+	StoreType      StoreResourceType  `yaml:"StoreResourceType"`
+	StoreResource  string             `yaml:"storeResource"`
```

Comment:
how to customize key names to consistent with code?

Reply:
to keep a simple experience, we may not add it in this pr. 
while we will track it as a todo feature, and support it if users have requests in the future